src/browser/main.coffee

```diff
@@ -12,6 +12,8 @@ yargs = require 'yargs'
 console.log = require 'nslog'
 
 start = ->
+  args = parseCommandLine()
```

Comment:
Does this need to be moved here in this pull request? Or could it go back to under the `app.setAppUserModelId('com.squirrel.atom.atom')` line?